benchmark/test/test_script.py

```diff
@@ -41,12 +41,13 @@ def capture_stderr():
 ''' % _YAML_CONTROL_SERVICE_ADDRESS
 
 
-class ClusterConfigurationTests(SynchronousTestCase):
+class ClusterConfigurationTests(TestCase):
     """
     Tests for mapping cluster configuration to cluster.
     """
 
     def setUp(self):
+        super(ClusterConfigurationTests, self).setUp()
```

Comment:
Oh, no, this is a TestCase rather than AsyncTestCase.

I wonder why we didn't call our base class `SynchronousTestCase`?